dags/deletes.py

```diff
@@ -369,19 +359,19 @@ def load_pending_person_deletions(
         )
 
         if len(current_chunk) >= chunk_size:
-            client.execute(
-                create_pending_person_deletions_table.populate_query,
-                current_chunk,
-            )
+            cluster.any_host_by_role(
+                Query(create_pending_person_deletions_table.populate_query, current_chunk),
+                NodeRole.DATA,
```

Comment:
This matches the input here: https://github.com/PostHog/posthog/blob/31535ab1dd03ac422c8351085e2e34b86c2f1b40/dags/deletes.py#L315-L316